src/components/include/application_manager/hmi_capabilities.h

```diff
@@ -95,6 +95,19 @@ class HMICapabilities {
    */
   virtual void set_ccpu_version(const std::string& ccpu_version) = 0;
 
+  /**
+   * @brief Interface that checks for compliance new software version of the
+   * target with last received
+   * @param ccpu_version Received system/hmi software version
+   */
+  virtual bool matches_ccpu_version(const std::string& ccpu_version) const = 0;
+
+  /**
+   * @brief Interface that update capabilities depending on ccpu_version
+   * @param ccpu_version Received system/hmi software version
+   */
+  virtual void on_software_version_received(const std::string& ccpu_version) = 0;
```

Comment:
Function does not looks like setter move to such functions and make camel case

Reply:
@LuxoftAKutsan done